tests/testsuite/lints/implicit_features/warn/mod.rs

```diff
@@ -27,10 +27,10 @@ implicit-features = "warn"
         .build();
 
     snapbox::cmd::Command::cargo_ui()
-        .masquerade_as_nightly_cargo(&["always_nightly"])
+        .masquerade_as_nightly_cargo(&["cargo-lints", "edition2024"])
         .current_dir(p.root())
         .arg("check")
-        .arg("--quiet")
+        .arg("-Zcargo-lints")
```

Comment:
Unrelated?

Reply:
This was needed to not show the warning about using `lints.cargo`

Reply:
Is that lint hidden with `--quiet` and so by removing that we now need this?

Shouldn't we have not gotten any lints without this `-Z`?